### PR TITLE
Assessment with explainer

### PR DESCRIPTION
**Pull request by DevChima**
Journey: 
https://whatsapp.turn.io/app/stacks/c14527da-bddb-4dcb-b6ff-6815b3bb3e85/8a4a887d-d60e-43e7-98d3-0e02c76d91a9

### CODE REVIEW

File: Assessments/form_journey_with_explainer.md [hunk 1]
```diff
@@ -0,0 +1,229 @@
...
+
+<!--
+ dictionary: "config"
+version: "0.1.0"
+columns: [] 
+-->
+
+| Key             | Value                                    |
+| --------------- | ---------------------------------------- |
+| api_token       | 22bbdd2a426526b55df8b3ed77eaa3523acfc6e7 |
```
Comment (HawkiesZA) [inline Assessments/form_journey_with_explainer.md]: Please remove the API Key. I need to revoke it for Onboarding as well :sweat_smile:

**Review by HawkiesZA — CHANGES_REQUESTED** [review]
We don't have to make new files each time, we can just update assessments.md and assessments.json.

File: Assessments/form_journey_with_explainer.md (hunk 1 above)
Comment (HawkiesZA) [reply Assessments/form_journey_with_explainer.md]: I flagged this, but looks like I made the mistake too. Doh!

Comment (DevChima) [reply Assessments/form_journey_with_explainer.md]: Lol. I didn't even notice it

Comment (DevChima) [reply Assessments/form_journey_with_explainer.md]: Ready for another look @HawkiesZA

**Review by HawkiesZA — APPROVED** [review]

**Pull request merged**